src/jsMain/kotlin/net/kyori/adventure/webui/js/Main.kt

```diff
@@ -494,16 +495,6 @@ private fun onWebsocketReady() {
         }
 }
 
-private fun onWebsocketClose() {
-    // We no longer have a working websocket connection, so any input changes would not go through. Display a little
-    // warning to the user and disable the input box to bring more attention to it, since changing the input would
-    // have no effect at this point anyway.
-    val warning = document.element<HTMLTextAreaElement>("connection-lost-warning")
-    val inputBox = document.element<HTMLTextAreaElement>("input")
```

Comment:
Should remove the HTML for this too

Reply:
This still makes sense to have in cases where, for example the user has lost internet connection altogether, where there should be some evidence that the reconnection is being attempted